assets/scss/_tech-partner.scss

```diff
@@ -0,0 +1,238 @@
+.tech-partner__heading {
+  background: linear-gradient(114.84deg, #133E9C -22.37%, #00091E 63.82%);
+  .tech-partner__headline {
+    @media only screen and (max-width: 738px) {
+      text-align: center;
+    }
+    .tech-partner__headline-bg {
+      background-image: url(../img/internet-future.png);
+      background-repeat: no-repeat;
+      background-position: 100% 90px;
+      background-size: 650px;
+      .col-sm-8 {
+        @media only screen and (max-width: 600px) {
+          margin: 0 auto;
+        }
+      }
+      .tech-partner__cta-button-wrapper {
+        @media only screen and (max-width: 768px) {
+          text-align: left;
+        }
+        @media only screen and (max-width: 600px) {
+          text-align: center;
+        }
+      }
+      @media only screen and (max-width: 991px) {
+        background-position: 100% 240px;
+        background-size: 550px;
+      }
+      @media only screen and (max-width: 600px) {
+        background-position: 100% 480px;
+        background-size: 550px;
+        height: 850px;
+      }
+
+      @media only screen and (max-width: 500px) {
+        background-position: 100% 590px;
+        background-size: 350px;
+        height: 850px;
+      }
+   }
+   h1, p {
+     color: #fff;
+     width: 150%;
+     text-align: left;
+     @media only screen and (max-width: 600px) {
+       width: auto;
+       text-align: center;
+     }
+   }
+   a {
+     .btn-signup {
+        font-size: 16px;
+        line-height: 24px;
+        background: #2683ff;
+        color: #fff;
+        border-radius: 6px;
+        width: 271px;
+        height: 48px;
+        border: 0;
+        outline: 0;
+      }
+    }
+  }
+  @media only screen and (max-width: 768px) {
+    background: radial-gradient(236.22% 100.97% at 50% -4.33%, #0059D1 0%, #000000 100%);
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  }
+}
+
+.tech-partner__partnership-types {
+  font-size: 18px;
+  h2 {
+    font-size: 28px;
+    bottom: 20px;
+    position: relative;
+    @media only screen and (max-width: 768px) {
+      bottom: 30px;
+    }
+  }
+  h4,p {
+    font-size: 18px;
+  }
+  img {
+    margin-bottom: 10px;
+  }
+  .tech-partner__partnership-types-content {
+    .col-md-6 {
+      max-width: 939px;
+      height: 320px;
+      background: #FFFFFF;
+      box-shadow: 0px 4px 74px rgba(31, 49, 76, 0.25);
+      border-radius: 10px;
+      div {
+        width: 80%;
+        margin: 0 auto;
+        margin-top: 10%;
+      }
+    }
+  }
+  .tech-partner__partnership-types-content-two {
+    position: relative;
+    left: 30px;
+    @media only screen and (max-width: 768px) {
+      position: relative;
+      top: 30px;
+      left: 0;
+    }
+  }
+
+  .spacer125 {
```

Comment:
i think this should be in a `_utility.scss` file so it's more consistent across the project

Reply:
These are for responsive edits particular to each use case, not site wide changes. Let me know if thats ok!